concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -101,40 +94,87 @@ def report_computed_task(timestamp = None, subtask_id = None, task_to_compute =
         )
 
 
+def create_task(task_to_compute_timestamp, task_id, compute_task_def_deadline):
+    return task_to_compute(
+        timestamp=task_to_compute_timestamp,
+        compute_task_def=compute_task_def(
+            task_id=task_id,
+            subtask_id=task_id,
+            deadline=compute_task_def_deadline,
+        )
+    )
+
+
+ProtocolConstants = namedtuple("ProtocolConstants",
+                               ["concent_messaging_time",
+                                "subtask_verification_time",
+                                "force_acceptance_time",
+                                "token_expiration_time"])
+
+
+def get_protocol_constants(cluster_url):
+    url = f"{cluster_url}/api/v1/protocol-constants/"
+    resp = requests.get(url)
+    json = resp.json()
+    concent_messaging_time = json['concent_messaging_time']['value']
+    subtask_verification_time = json['subtask_verification_time']['value']
+    force_acceptance_time = json['force_acceptance_time']['value']
+    token_expiration_time = json['token_expiration_time']['value']
+    constants = ProtocolConstants(concent_messaging_time, subtask_verification_time, force_acceptance_time,
```

Comment:
```
constants = ProtocolConstants(
    concent_messaging_time,
    subtask_verification_time, 
    force_acceptance_time,
    token_expiration_time,
)
```

Reply:
Also, with lists like this, I'd use named parameters because the order is not really set in stone and if someone changes it, it may be hard to notice that the times are now wrong.

``` python
constants = ProtocolConstants(
    concent_messaging_time    = concent_messaging_time,
    subtask_verification_time = subtask_verification_time, 
    force_acceptance_time     = force_acceptance_time,
    token_expiration_time     = token_expiration_time,
)
```

Or, to make things shorter with the same effect:
``` python
constants = ProtocolConstants(**json)
```

Reply:
@cameel, `constants = ProtocolConstants(**json)` might be unsafe if json contains additional params